.github/workflows/unit_tests.yml

```diff
@@ -61,4 +61,24 @@ jobs:
     - name: Cleaning
       if: ${{ always() }}
       run: ./scripts/clean_fcctr.sh
+
+  profile-test:
+    name: "Unit tests: profile submodule"
```

Comment:
you should merge in your test into the first job (Line 15). It should runs all modules' test

Reply:
Done